www/core/nightlies/next_minor_list.xml

```diff
@@ -1,6 +1,7 @@
 <extensionset name="Joomla! Core Nightly Builds" description="Joomla! Core Next Minor Nightly Builds">
 	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="4.4.9" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="4.4.10" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="5.3.0-alpha4-dev" targetplatformversion="4.4.11" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
```

Comment:
Maybe we should have one line in addition with targetplatformversion="4.4.12" to it matched the 4.4.12-dev of the 4.4 nightly builds and people can update from such a nightly to 5.3?

Reply:
Yes line added, change to 5.3.0-beta1-dev